qp_klp/klp.py

```diff
@@ -72,25 +72,35 @@ def sequence_processing_pipeline(qclient, job_id, parameters, out_dir):
     bool, list, str
         The results of the job
     """
-    # Assume that for a job to be considered a restart, there must be work
-    # performed worth re-starting for. Since the working directory for each
-    # step is created only if the previous steps were successful, testing
-    # for the presence of them ensures that n-1 steps exist and were
-    # successful.
-
-    # at minimum, ConvertJob needs to have been successful.
+    status_line = StatusUpdate(qclient, job_id)
+
+    # NB: RESTART FUNCTIONALITY:
+    # To restart a job that's failed, simply create a ProcessingJob() object
+    # in Qiita's interpreter using the Qiita Job ID aka the name of the
+    # working directory. _set_status("in construction") and then call
+    # submit(). New SPP jobs will record all the information needed to perform
+    # a restart should it fail.
     out_path = partial(join, out_dir)
-    is_restart = True if exists(out_path('NuQCJob')) else False
+    is_restart = True if exists(out_path('restart_me')) else False
```

Comment:
It's a good change actually. Since the user isn't supplying the three required parameters for restart and we previously weren't saving them, we had to scour the files in the working directory for them. This worked when we could extract the run-id from ConvertJob's sbatch file. This doesn't work for TellRead however.

Now every new job submission saves its parameters in 'restart_me' and if the job is restarted, klp can definitively say this is a restart or not.